postmodern/tests/test-dao.lisp

```diff
@@ -8,142 +8,172 @@
+  (:keys id))
 
 (defclass test-data-short ()
   ((id :col-type serial :initarg :id :accessor test-id)
    (a :col-type (or (varchar 100) db-null) :initarg :a :accessor test-a))
-  (:metaclass dao-class)
+  (:metaclass postmodern:dao-class)
   (:table-name dao-test)
   (:keys id))
 
 (defclass test-data-short-wrong-1 ()
   ((id :col-type serial :initarg :id :accessor test-id)
    (a :col-type (or numeric db-null) :initarg :a :accessor test-a))
-  (:metaclass dao-class)
+  (:metaclass postmodern:dao-class)
   (:table-name dao-test)
   (:keys id))
 
 (defclass test-data-d-string ()
-    ((id :col-type serial :initarg :id :accessor test-id)
-     (a :col-type (or (varchar 100) db-null) :initarg :a :accessor test-a)
-     (b :col-type boolean :col-default nil :initarg :b :accessor test-b)
-     (c :col-type integer :col-default 0 :initarg :c :accessor test-c)
-     (d :col-type text :col-default "" :initarg :d :accessor test-d))
-    (:metaclass dao-class)
-    (:table-name dao-test)
-    (:keys id))
+  ((id :col-type serial :initarg :id :accessor test-id)
+   (a :col-type (or (varchar 100) db-null) :initarg :a :accessor test-a)
+   (b :col-type boolean :col-default nil :initarg :b :accessor test-b)
+   (c :col-type integer :col-default 0 :initarg :c :accessor test-c)
+   (d :col-type text :col-default "" :initarg :d :accessor test-d))
+  (:metaclass postmodern:dao-class)
+  (:table-name dao-test)
+  (:keys id))
 
 (test dao-class
-  (with-test-connection
-    (when (table-exists-p 'dao-test)
-      (query (:drop-table :if-exists 'dao-test :cascade)))
-    (is (not (table-exists-p 'dao-test)))
-    (is (equal (dao-table-definition 'test-data)
-               "CREATE TABLE dao_test (id SERIAL NOT NULL, a VARCHAR(100) DEFAULT NULL, b BOOLEAN NOT NULL DEFAULT false, c INTEGER NOT NULL DEFAULT 0, d NUMERIC NOT NULL DEFAULT 0.0, PRIMARY KEY (id))"))
-    (is (equal (dao-keys (find-class 'test-data))
-               '(ID)))
-    (dao-keys (make-instance 'test-data :id 1))
-    '(1)
-    (execute (dao-table-definition 'test-data))
-    (is (table-exists-p 'dao-test))
-    (protect
-      (is (member :dao-test (list-tables)))
-      (is (null (select-dao 'test-data)))
-      (let ((dao (make-instance 'test-data :a "quux")))
-        (signals error (test-id dao))
-        (insert-dao dao)
-        (is (dao-exists-p dao))
-        (let* ((id (test-id dao))
-              (database-dao (get-dao 'test-data id)))
-          (is (not (null database-dao)))
-          (is (eql (test-id dao) (test-id database-dao)))
-          (is (string= (test-a database-dao) "quux"))
-          (setf (test-b dao) t)
-          (update-dao dao)
-          (let ((new-database-dao (get-dao 'test-data id)))
-            (is (eq (test-b new-database-dao) t))
-            (is (eq (test-b database-dao) nil))
-            (delete-dao dao))))
-      (is (not (select-dao 'test-data)))
-      (execute (:drop-table 'dao-test :cascade)))))
+      (with-test-connection
```

Comment:
These `test` forms seem misindented now. Doesn't `test` have a `&body` which should indent these with two spaces instead of like a function?

Reply:
Hmm. Emacs is not changing the indentation for me. Will look further.

Reply:
Still puzzled on why emacs is doing this, but used C-M-q to semi-manually fix this.